asus-ec-sensors.c

```diff
@@ -144,6 +144,10 @@ enum ec_sensors {
 	ec_sensor_temp_sensor_extra_2,
 	/* "Extra_3" temperature sensor reading [℃] */
 	ec_sensor_temp_sensor_extra_3,
+	/* USB4 fan [RPM] */
```

Comment:
You might want to put these next to the other fans, order is unimportant in this enum, as far as I remember.

Reply:
Done